cob/cli/docker_cli.py

```diff
@@ -425,9 +425,13 @@ def run_image(image_name, detach, compose_overrides, http_port):
 
 @docker.command(name="stop-image")
 @click.argument('image_name')
-def stop_image(image_name):
+@click.option('compose_overrides', '-o', '--overlay-compose-file', default=[], multiple=True)
```

Comment:
It's not recommended to have mutable objects as defaults in Python

Reply:
I just went with the same lines as in https://github.com/getweber/cob/blob/f2a5d74a15f5262d7980e4cf1f1a20af29194ffb/cob/cli/docker_cli.py#L412
I guess I can change it to a tuple then.

Reply:
Done